chain-tx-enclave-next/enclave-ra/ra-client/src/verifier.rs

```diff
@@ -108,7 +108,8 @@ impl EnclaveCertVerifier {
             return Err(EnclaveCertVerifierError::CertificateExpired);
         }
 
-        let attestation_report_oid = Oid::from(OID_EXTENSION_ATTESTATION_REPORT);
+        let attestation_report_oid = Oid::from(OID_EXTENSION_ATTESTATION_REPORT)
+            .expect("Unable to parse attestation report OID");
```

Comment:
so the parser panics if one constructs a malicious cert? 
extending and returning `EnclaveCertVerifierError` seems safer?

Reply:
This is just creating an Object ID. `OID_EXTENSION_ATTESTATION_REPORT` is a compile time constant.

Reply:
ok that's all right